t/security.t

```diff
@@ -0,0 +1,61 @@
+use Mojo::Base -strict;
+use Test::Mojo;
+use Test::More;
+
+use Mojolicious::Lite;
+post '/protected' => sub {
+  my $c = shift->openapi->valid_input or return;
+  $c->reply->openapi(200 => {ok => 1});
+  },
+  'protected';
+
+plugin OpenAPI => {
+  url      => 'data://main/sec.json',
+  security => {
+    dummy => sub {
+      my ($c, $config, $next) = @_;
+      return $c->$next if $c->req->headers->authorization;
+      return $c->render(json => $config, status => 401);
```

Comment:
I kind of like the idea of calling `$next` or needing to render a document.

Need to fix typo. Should be:

```
return $c->render(openapi => $config, status => 401);
```